src/dcap_provider.cpp

```diff
@@ -54,12 +54,15 @@ static const std::map<std::string, std::string> default_values = {
 // New API version used to request PEM encoded CRLs
 constexpr char API_VERSION_LEGACY[] = "api-version=2018-10-01-preview";
 constexpr char API_VERSION[] = "api-version=2020-02-12-preview";

```

Comment:
```suggestion
constexpr char API_VERSION[] = "api-version=2021-07-22-preview";
```

Reply:
Scratch that. I think what you might want to do is: 
`constexpr char API_VERSION_THIM_AGENT[] = "api-version=2021-07-22-preview";`

You can use the new API version for the GetCert calls, and the old API version for the other collateral.